### DISCUSSION TITLE
New github organisation for Open FEDEM

### DISCUSSION THREAD

**Discussion by kmokstad** [openfedem/fedem-gui#3] [Announcements]
Dear contributors, @sanguinariojoe, @freeNastran, @core-marine-dev, @norcompute.

As you may have noticed the Open FEDEM source repositories that you have showed some interest in have been archived and set to read-only under the SAP-archive github organisation. However, we have now forked all of them under the new [openfedem](https://github.com/openfedem) organisation, and the maintenance of the open source project will continue from there. We therefore encourage you to update your forks to point to these repositories instead.

Thanks,
The Open FEDEM team

**Comment by freeNastran** [comment]
If the old one is archived and the new one will not merge into the old one, I suggest breaking the fork. Don't fork it, but create the new one and upload from local file. Otherwise you will always see that info: 
`This branch is [20 commits ahead of](https://github.com/openfedem/fedem-gui/compare/SAP-archive%3Afedem-gui%3Amain...main), [2 commits behind](https://github.com/openfedem/fedem-gui/compare/main...SAP-archive%3Afedem-gui%3Amain) SAP-archive/fedem-gui:main.`

**Reply by kmokstad** [reply]
Yes, I noticed this myself also, and don't like it too much either. Another "problem" is that whenever you open a pull request, it'll by default suggest to merge with the original, archived, repository and not the forked one, and you have to manually change this as well.

I think it should be possible to break the fork while retaining existing issues, PRs and discussions, as suggested for instance [here](https://stackoverflow.com/questions/16052477/delete-fork-dependency-of-a-github-repository) . I'll explore this further over the summer, but we'll need to make sure doing this is fine for the Copyright holder also (SAP) before changing this.
